### PR TITLE
Rebased version of #64

### PR DESCRIPTION
**Pull request by pcahyna**
see #64

### CODE REVIEW

**Comment by pcahyna** [comment]
this fails with "volume 'luks-vdb1' cannot be resized from 4 GiB to 4 GiB".

File: library/blivet.py [hunk 1]
```diff
@@ -325,6 +325,8 @@ def _resize(self):
                 raise BlivetAnsibleError("volume '%s' cannot be resized from %s to %s: %s" % (self._device.name,
                                                                                               self._device.size,
                                                                                               size, str(e)))
+        elif size and self._device.size != size and not self._device.resizable:
```
Comment (yizhanglinux) [inline library/blivet.py:328]: Hi
I'm trying to cover this line, could you give me more info about on what condition the device will not be resizeable?

Thanks
Yi

**Comment by yizhanglinux** [comment]
> Hi
> I'm trying to cover this line, could you give me more info about on what condition the device will not be resizeable

Pls ignore this comment, I cannot found the delete option for this comment.

**Comment by yizhanglinux** [comment]
Sorry, closed it by mistake.

File: library/blivet.py (hunk 1 above)
Comment (pcahyna) [reply library/blivet.py:328]: >  I cannot found the delete option for this comment.

Click the three dots in the top right corner?

Comment (pcahyna) [reply library/blivet.py:328]: > I'm trying to cover this line, could you give me more info about on what condition the device will not be resizeable?

Are you trying to test the problems that this commit fixes, or those that it introduces? The latter is a test failure  "volume 'luks-vdb1' cannot be resized from 4 GiB to 4 GiB". Maybe wait for developers to fix it before developing tests?

**Comment by pcahyna** [comment]
@dwlehman @japokorn can you please look at the test failure "volume 'luks-vdb1' cannot be resized from 4 GiB to 4 GiB" and fix the problem, so that @yizhanglinux or others can continue developing tests for problems that this PR is supposed to fix?

File: library/blivet.py (hunk 1 above)
Comment (yizhanglinux) [reply library/blivet.py:328]: > > I cannot found the delete option for this comment.
> 
> Click the three dots in the top right corner?

I tried, but no that option. )
no-delete

Comment (yizhanglinux) [reply library/blivet.py:328]: > > I'm trying to cover this line, could you give me more info about on what condition the device will not be resizeable?
> 
> Are you trying to test the problems that this commit fixes, or those that it introduces? The latter is a test failure "volume 'luks-vdb1' cannot be resized from 4 GiB to 4 GiB". Maybe wait for developers to fix it before developing tests?

I'm trying to test the the problems that this commit fixed.
OK, will skip this commit.

**Comment by dwlehman** [comment]
See #125.

File: library/blivet.py (hunk 1 above)
Comment (pcahyna) [reply library/blivet.py:328]: @yizhanglinux The test failure "volume 'luks-vdb1' cannot be resized from 4 GiB to 4 GiB" is now fixed in #125 , can you please continue working on that PR?

**Comment by pcahyna** [comment]
@dwlehman thanks, closing this PR as it is superseded by #125.

**Pull request closed without merging**